tools/toolchain/optimized_clang.sh

```diff
@@ -160,14 +162,19 @@ if [[ "${CLANG_BUILD}" = "INSTALL" ]]; then
     cmake -B build -S llvm "${CLANG_OPTS[@]}" -DLLVM_PROFDATA_FILE="$(realpath combined.prof)" -DCMAKE_EXE_LINKER_FLAGS="-Wl,--emit-relocs"
     ninja -C build
 
+    mkdir -p "${CLANG_SYSROOT_DIR}"
+    DESTDIR="${CLANG_SYSROOT_DIR}" ninja -C build install-distribution-stripped
     cd "${CLANG_ROOT_DIR}"
-    rm -rf "${CLANG_BUILD_DIR}"/{build/profiles,*.prof,prof.fdata}
-    tar -cpzf "${CLANG_ARCHIVE}" "${CLANG_CHECKOUT_NAME}"
+    tar -cpzf "${CLANG_ARCHIVE}" "${CLANG_SYSROOT_NAME}"
```

Comment:
I think it's better to use `-C "${CLANG_SYSROOT_NAME}"`, then you get a tarball with /usr/local/{all-the-files} instead of the artificial CLANG_SYSROOT_NAME. Users can then unpack it in their root directory so it works outside dbuild too (I did this for the old version too, but this is easier).